src/bflib_sound.c

```diff
@@ -730,6 +732,32 @@ void play_non_3d_sample_no_overlap(long smpl_idx)
     }
 }
 
+void play_atmos_sound(long smpl_idx)
```

Comment:
Indent is messed up below.

Reply:
I'm looking at it, but I don't know what is wrong. Could you please elaborate what you suggest I change?

Reply:
But they do? Line 737, 739, 741 and 742 all have the exact same indent. Both on the local source file and here on github.

Reply:
I see that in the review screen that is not the case, but look here:
https://github.com/Loobinex/keeperfx-unofficial/blob/ManualSoundFont/src/bflib_sound.c#L737